implementations/elixir/ockam/ockam/lib/ockam/wire.ex

```diff
@@ -12,7 +12,7 @@ defmodule Ockam.Wire do
   require DecodeError
   require EncodeError
 
-  @default_implementation Ockam.Wire.Binary.V1
+  @default_implementation Ockam.Wire.Binary.Versions
```

Comment:
I realize that I probably started us down this path, but in thinking through how this needs to be deployed now, and how a version 3 might work, I believe this should remain V1, the file v1.ex should be reverted to before I changed it, and that the majority of the changes we made should be in a v2.